Library/Homebrew/keg.rb

```diff
@@ -208,6 +208,7 @@ def ==(other)
 
   def empty_installation?
     Pathname.glob("#{path}/**/*") do |file|
+      return false if file.executable?
       next if file.directory?
       basename = file.basename.to_s
       next if Metafiles.copy?(basename)
```

Comment:
Sorry! I mean instead of using `Pathname.glob("#{path}/**/*")` above make check for metafiles just the root directory of the keg (which is the prefix) rather than in every subdirectory (i.e. `bin` in your case).